packages/components/src/confirm-dialog/component.tsx

```diff
@@ -75,8 +77,8 @@ function ConfirmDialog(
 		[ handleEvent, onConfirm ]
 	);
 
-	const cancelLabel = __( 'Cancel' );
-	const confirmLabel = __( 'OK' );
+	const cancelLabel = cancelButtonText ? cancelButtonText : __( 'Cancel' );
+	const confirmLabel = confirmButtonText ? confirmButtonText : __( 'OK' );
```

Comment:
Non-blocking: Maybe better for readability?

```suggestion
	const cancelLabel = cancelButtonText ?? __( 'Cancel' );
	const confirmLabel = confirmButtonText ?? : __( 'OK' );
```